tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1090,6 +1093,65 @@ bool FindMatMulBiasAddAndGelu(RemapperContext* ctx, int node_index,
   return (found_gelu_exact || found_gelu_approximate);
 }
 
+bool FindSigmoidAndMul(RemapperContext* ctx, int node_index,
+                       std::map<string, int>* matched_nodes_map,
+                       std::set<int>* remove_node_indices) {
+  using utils::MatchingDirection;
+  using utils::NodeStatus;
+  // clang-format off
+  //                Convert Sigmoid+Mul to Swish
+  //          From Graph                          To Graph
+  //          -----------                         ---------
+  //    Conv2D  <-  Filter(const)           Conv2D  <-  Filter(const)
+  //      !                                   !
+  //      V                                   V
+  //    BiasAdd <-  bias(const)             BiasAdd <-  bias(const)
+  //      !                                   !
+  //      V                                   !
+  //  ---- ----                               !
+  //  !       !                               !
+  //  !       V                               !
+  //  !    Sigmoid                            !
+  //  !       !                               !
+  //  ---   ---                               !
+  //     !  !                                 !
+  //     !  !                                 !
+  //     V  V                                 V
+  //      Mul                             _MklSwish
+  //      !                                   !
+  //      V                                   V
+
```

Comment:
The Conv2D and BiasAdd part is unrelated to the fusion. I think we can just use the one-liner comment in your PR description here (`Mul(x, Sigmoid(x)) --> Swish(x)`.

Reply:
Done